testing/pavucontrol/APKBUILD

```diff
@@ -0,0 +1,36 @@
+# Contributor: Fredrick Brennan <copypaste@kittens.ph>
+# Maintainer: Fredrick Brennan <copypaste@kittens.ph>
+pkgname=pavucontrol
+pkgver="20171007"
```

Comment:
we try avoid making releases of random git snapshots. Instead, we should use releases from upstream. In this case the latest release is 3.0.

Also, if 4.0 comes out, apk will consider the number `20171007` bigger than `4` so it will not upgrade.

Reply:
I tried that first actually. The current release of `pavucontrol` is very old and no longer works correctly with the `pulseaudio` package. What should I do?

Reply:
Contact upstream and convince them to finally release a new version.

Meanwhile you can use version like `3.0_git20171007`. Or upstream tell you that the next release will increment _major_, it may be better to use `4.0_pre20171007`.